src/module/chat-message/listeners/set-as-initiative.ts

```diff
@@ -29,7 +29,12 @@ export const SetAsInitiative = {
             setInitiativeButton.title = game.i18n.localize("PF2E.ClickToSetInitiative");
             setInitiativeButton.appendChild(fontAwesomeIcon("fa-swords", { style: "solid" }));
             btnContainer.appendChild(setInitiativeButton);
-            li.querySelector(".dice-total")?.appendChild(btnContainer);
+            if (message.isReroll) {
+                li.querySelector(".pf2e-reroll-discard .dmgBtn-container.dice-total-setInitiative-btn")?.remove();
```

Comment:
Does anything need to be removed? The set-as-initiative button is added dynamically.

Reply:
Nope, just me overcomplicating it. Thanks!